unstructured/ingest/ingest_doc_json_mixin.py

```diff
@@ -0,0 +1,54 @@
+import json
+import typing as t
+from pathlib import Path
+
+from dataclasses_json import DataClassJsonMixin
+from dataclasses_json.core import _ExtendedEncoder
+
+
+class IngestDocJsonMixin(DataClassJsonMixin):
```

Comment:
docstring explaining what/why

i.e. adds in all the @property fields to the serialized json already being created via the DataClassJsonMixin...etc